src/main/kotlin/gg/skytils/skytilsmod/utils/ItemUtil.kt

```diff
@@ -29,6 +29,7 @@ import kotlin.math.max
 
 object ItemUtil {
     private val PET_PATTERN = "§7\\[Lvl \\d+] (?<color>§[0-9a-fk-or]).+".toRegex()
+    private val PET_PATTERN_FOR_SYNTHESIS = "§7\\[\\d+] (?<color>§[0-9a-fk-or]).+".toRegex()
```

Comment:
We won't be adding support for non-hypixel values.

Reply:
got it.